lib/matplotlib/backends/qt_editor/figureoptions.py

```diff
@@ -235,15 +235,26 @@ def apply_callback(data):
 
         # re-generate legend, if checkbox is checked
         if generate_legend:
-            draggable = None
-            ncol = 1
+            kwargs = {'ncol' : 1}
```

Comment:
I think we should start with `kwargs = {}` as the default value of ncol is 1 anyway (and it solves the flake8 issue).